src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25598.cs

```diff
@@ -0,0 +1,28 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25598 : _IssuesUITest
+	{
+		public Issue25598(TestDevice testDevice) : base(testDevice) { }
+
+		public override string Issue => "IndicatorView with Template won't show when ItemSource reaches 0 Elements";
+
+		[Test]
+		[Category(UITestCategories.IndicatorView)]
+		public void IndicatorWithTemplateShouldBeVisible()
+		{
+			App.WaitForElement("RemoveItemButton");
+
+			for (int i = 0; i < 3; i++)
+				App.Click("RemoveItemButton");
+
+			for (int i = 0; i < 3; i++)
+				App.Click("AddItemButton");
+
+			VerifyScreenshot();
```

Comment:
Snapshots already available in the latest build.

Example, iOS:
![image](https://github.com/user-attachments/assets/3b7520d0-b48f-43b3-bf9b-7e4694a0e1b8)